### ISSUE TITLE
Failure on run

### ISSUE THREAD

**Issue by mcepl** [robodair/pydocstring#3]
I have cloned your repository and then run ``python setup.py develop --user``. Then I switched to the repository with my project (it is [M2Crypto](https://gitlab.com/m2crypto/m2crypto/), BTW) and then this happened:
```
 $ pydocstring -f reST M2Crypto/BIO.py
Traceback (most recent call last):
  File "/home/matej/.local/bin/pydocstring", line 9, in <module>
    load_entry_point('pydocstring==0.0.dev14', 'console_scripts', 'pydocstring')()
  File "/home/matej/archiv/knihovna/repos/tmp/pydocstring/pydocstring/cli.py", line 42, in main
    print('"""\n' + output + '"""\n')
TypeError: cannot concatenate 'str' and 'NoneType' objects
 $
```
Using python 2.7 on RHEL-7 (if it matters, the same happened with python 3.4.5).

**Comment by robodair** [comment]
What were the arguments it was run with, if any?

**Comment by robodair** [comment]
Oh hang on, you called it with the path to a file. Sorry I didn't read that properly.

Currently you need to give it the actual source, I'll add a check if it's a file and instead read the contents.

**Comment by robodair** [comment]
ebd4a73678d0e84f5603a50465a77adbe08e11f3 should let you give a file path, but docstrings aren't added in-place to the file, a docstring is generated for the given position of the cursor and printed out.

**Comment by mcepl** [comment]
> Oh hang on, you called it with the path to a file. Sorry I didn't read that properly.

Then the ``--help`` is misleading, I am afraid. So, the code should go to ``STDIN``?

**Comment by robodair** [comment]
The source of the code was the first argument, e.g:

    pydocstring "def test(p1, p2) -> int:" -f numpy

I've changed it to allow giving a filepath in the form you used it, but note that docstrings aren't replaced in place - they're just generated for the function/class/module that the `position` argument places the cursor inside.

You might want to use it like this:

    pydocstring -f numpy BIO.py <cursor position>

But I've just tried that and it doesn't work, but the autocompletion of the docstring does work from within vscode. I'll have a look into why.
